spec/factories/supervisors.rb

```diff
@@ -5,7 +5,8 @@
 
     trait :with_casa_cases do
       after(:create) do |user, _|
-        create_list(:case_assignment, 2, volunteer: user)
+        volunteer = create(:volunteer)
```

Comment:
the user in this case is I believe the supervisor. The cases should not be assigned to the supervisor.